# robotModelId == 1 => MILDRobotModelWithExactIK
# robotModelId == 2 => MILDRobotModelWithApproximatedIK
robotModelId: 1
debugLevels: [NONE] # ALL, NONE, PARAMETERS, RATING, ROBOT_MODEL, MAP, IK_RATING
panMin: -20
panMax: 20
tiltMin: -45
tiltMax: 25
mSigma: 2.0
colThresh: 40 # greater values results in problems during simulation in navigation. Smaller -> more distance to obstacles.

# Parameters only used for the MILDRobotModelWithApproximatedIK
mOmegaPan: 1  # weighting factor for rotating the pan-axis of the PTU
mOmegaTilt: 1 # weighting factor for rotating of the tilt-axis of the PTU
mOmegaRot: 10 # weighting factor for rotating the base
mOmegaBase: 135 # weighting factor for moving the base
tolerance: 0.00003 # tolerance for pathfinding of global planer

# Parameters only used for the MILDRobotModelWithApproximatedIK
panAngleSamplingStepsPerIteration: 20
inverseKinematicIterationAccuracy: 0.005
IKAngleRating: ANGLE_APPROXIMATION #The rating algorithm used for the orientation angle sampling in the MILDRobotModelWithExactIK module. Available algorithms are: ANGLE_APPROXIMATION, NAVIGATION_PATH, SIMPLE
visualizeIK: true
useGlobalPlanner: false
ncp: 0.4 # distance to NearClippingPlane in meter
fcp: 1.5 # distance to FarClippingPlane in meter


